exonum-light-client/src/main/java/com/exonum/client/ExonumHttpClient.java

```diff
@@ -145,7 +151,16 @@ public BlockResponse getBlockByHeight(long height) {
     Map<String, String> query = ImmutableMap.of("height", String.valueOf(height));
     Request request = get(url(BLOCK, query));
 
-    return blockingExecuteAndParse(request, ExplorerApiHelper::parseGetBlockResponse);
+    return blockingExecute(request, response -> {
```

Comment:
Why not move this block to a function if it's duplicated?

Reply:
It is not duplicated as-is, so could you please suggest a viable abstraction?

Reply:
Sorry, didn't notice the difference between these two - it's all good then.